Worker.php

```diff
@@ -1511,8 +1518,19 @@ public function stop()
     public function acceptConnection($socket)
     {
         // Accept a connection on server socket.
-        $new_socket = @stream_socket_accept($socket, 0, $remote_address);
-        // Thundering herd.
+        if($this->count>1 && $this->reusePort==false){
```

Comment:
Yes triple equals operator has better performance.

But this pull request will not be accepted, because we found no significant change in performance after testing.
I think the reason is that using flock has performance loss too which is not significantly lower than thundering herd.